src/python/ProcInfo.py

```diff
@@ -286,9 +286,9 @@ def diffWithOverflowCheck(this, new, old):
 		if new >= old:
 			return new - old;
 		else:
-			max = (1L << 31) * 2;  # 32 bits
+			max = (1 << 31) * 2;  # 32 bits
 			if old >= max:
-				max = (1L << 63) * 2;  # 64 bits
+				max = (1 << 63) * 2;  # 64 bits
```

Comment:
This is the section of code I would be careful of. Make sure you test it. Because in python3 there is no difference between a long and an int.